packages/components/src/internal/app/utils.ts

```diff
@@ -113,6 +113,24 @@ export function isFreezerManagementEnabled(): boolean {
     );
 }
 
+export function isProductNavigationEnabled(productId: string): boolean {
+    const lcProductId = productId?.toLowerCase();
+    if (lcProductId === SAMPLE_MANAGER_PRODUCT_ID.toLowerCase()) {
```

Comment:
didn't the comment for the PRODUCT_ID const updates say that they were changed so that we didn't have to do toLowerCase() in comparisons anymore? Does that not apply here?